src/views/studio/lib/studio-member-actions.js

```diff
@@ -187,6 +187,26 @@ const acceptInvitation = () => ((dispatch, getState) => new Promise((resolve, re
     });
 }));
 
+const transferHost = (password, newHostName, newHostId) =>
+    ((dispatch, getState) => new Promise((resolve, reject) => {
+        const state = getState();
+        const studioId = selectStudioId(state);
+        const token = selectToken(state);
+        newHostName = newHostName.trim();
+        api({
+            uri: `/studios/${studioId}/transfer/${newHostName}?password=${password}`,
+            method: 'PUT',
+            authentication: token,
+            withCredentials: true,
+            useCsrf: true
+        }, (err, body, res) => {
+            const error = normalizeError(err, body, res);
+            if (error) return reject(error);
+            dispatch(setInfo({owner: newHostId}));
+            return resolve();
+        });
```

Comment:
Does this mean that the old host will still remain as a manager?

Reply:
@FunctionalMetatable yep!